server/jetstream_cluster.go

```diff
@@ -2918,6 +2918,9 @@ func (js *jetStream) processClusterCreateConsumer(ca *consumerAssignment, state
 		// Start our monitoring routine.
 		if rg.node == nil {
 			// Single replica consumer, process manually here.
+			js.mu.Lock()
+			ca.responded = false
```

Comment:
Maybe add quick comment, // to force response in case we think we have responded before.